tasks/tiler/xyz.py

```diff
@@ -34,24 +35,29 @@ def get_config_file(self):
     def get_geography_level(self, level):
         raise NotImplementedError('Geography levels file must be implemented by the child class')
 
-    def requires(self):
-        return {
-            'shorelineclip': ShorelineClip(geography=self.geography, year='2015')
-        }
-
     def run(self):
         config_data = self._get_config_data()
         for table_config in config_data:
             table_schema = self._get_table_schema(table_config)
             table_bboxes = table_config['bboxes']
             self._create_schema_and_table(table_schema)
-            self._generate_and_insert_tiles(self.zoom_level, table_schema, table_config['columns'], table_bboxes)
+            self._generate_and_insert_tiles(self.zoom_level, table_schema, table_config, table_bboxes)
+
+    def _get_table_name(self, table_schema):
+        return table_schema['table_name'] + '_' + self.month
+
+    def _get_mc_dates(self):
+        session = current_session()
+        query = '''
+                SELECT cdb_observatory.OBS_GetMCDates('{schema}', '{geo_level}', '{month}');
+                '''.format(schema='us.mastercard', geo_level=self.geography, month=self.month)
+        return session.execute(query).fetchone()[0]
 
     def _create_schema_and_table(self, table_schema):
         session = current_session()
         session.execute('CREATE SCHEMA IF NOT EXISTS tiler')
         cols_schema = []
-        table_name = "{}.{}".format(table_schema['schema'], table_schema['table_name'])
+        table_name = "{}.{}".format(table_schema['schema'], self._get_table_name(table_schema))
```

Comment:
We should change this to `"\"{}\".\"{}\""` in order to use double quote with the postgis schema or table

Reply:
Totally. On it